playbooks/azure/openshift-cluster/build_node_image.yml

```diff
@@ -55,6 +55,12 @@
 - name: install openshift
   import_playbook: ../../openshift-node/private/image_prep.yml
 
+- name: persist oreg_url
+  copy:
+    dest: "/root/oreg_url"
```

Comment:
/etc ?

Reply:
Maybe? I guess I can grab the file in acs-engine before untarring the config bundle...

Reply:
Oreg URL gets laid down in a few places already.  Check `/root/.docker/config.json`.  Also check `/etc/ansible/facts.d/`.  Ansible generally persists facts here.  I believe oreg_url should be there.  

If this is not found, then I might suggest a known location:

1. `/etc/origin/oreg_url`
2. `/etc/origin/config/oreg_url`